website/templates/project/project.mako

```diff
@@ -16,7 +16,7 @@
                         </h2>
                     % else:
                         <h2 class="node-parent-title unavailable">
-                            <span>Private Project</span>&nbsp;/
+                            <span>Private Project</span> <i class="fa fa-level-down fa fa-dark-lg"> </i>
```

Comment:
But this design has been in production for a while and should have gone through review before, so I feel like it's a bit late for this discussion. I don't have strong feelings either way but we need to be careful about changing the UI too often, especially if it could have been avoided with proper process.